docs/guide/nameservice/02-messages.md

```diff
@@ -107,17 +107,17 @@ The `ignite scaffold message buy-name name bid` command creates and modifies sev
     syntax = "proto3";
     package cosmonaut.nameservice.nameservice;
 
-    // this line is used by starport scaffolding # proto/tx/import
+    // this line is used by ignite scaffolding # proto/tx/import
```

Comment:
Can we avoid making changes to this one and similar ones.

Basically everything starts as `// this line is used by`, we shouldn't change the starport name there for backwards compatibility.